src/redturtle/prenotazioni/content/prenotazioni_folder.py

```diff
@@ -593,7 +613,24 @@ def data_validation(data):
         defaultFactory=notify_on_refuse_message_default_factory,
         required=False,
     )
-
+    notify_as_reminder_subject = schema.TextLine(
+        title=_(
+            "notify_as_reminder_subject",
+            default="Booking reminder subject.",
+        ),
+        description=_("notify_as_reminder_subject_help", default=""),
+        defaultFactory=notify_as_reminder_subject_default_factory,
+        required=False,
+    )
+    notify_as_reminder_message = schema.Text(
+        title=_(
+            "notify_as_reminder_message",
+            default="Booking reminder message.",
+        ),
+        description=_("notify_as_reminder_message", default=""),
```

Comment:
same..add more infos here

Reply:
The field name is aligned with other fields name style from the same tab.
Also due to volto rendering problems we do not use the help text in this tab

Reply:
i'm not complaining with naming convention, i want to add user more infos (who cares about volto)